docs/software/cloud.md

```diff
@@ -230,6 +230,40 @@ Nextcloud gives you access to all your files wherever you are. Store your docume
 
     Your user data directory will stay after deinstallation. As well a database backup will be saved to your user data directory. Thus you can easily restore your instance by reinstalling Nextcloud and restore the database dump.
 
+    **How can I check my OPcache status**
+
+    Nextcloud has a speedup option via the **PHP OPcache** feature. DietPi activates this during installation of Nextcloud (e.g. via `97-dietpi.ini` and `98-dietpi-nextcloud.ini`).  
+    If the OPcache is not setup in an appropriate manner, the Nextcloud settings overview dialog shows *"PHP OPcache not properly configured"* in combination with needed settings. If this should appear, an issue in <https://github.com/MichaIng/DietPi/issues> should be opened, because the DietPi installation shall do it all for the user in a correct manner.
+
+    Sometimes there is the question which of the various `php.ini` files on the system are used. DietPi therefore has helping files in its `/var/www` directory to give some information about this:  
+    The first file is `phpinfo.php`. This can be called from the browser via
+
+    - URL = `http://<your.IP>/phpinfo.php/`
+
+    In the following output there are some hints which files are used:
+
+    - `Configuration File (php.ini) Path`
+    - `Loaded Configuration File`
+    - `Scan this dir for additional .ini files`
+    - `Additional .ini files parsed`
+
+    The actual OPcache status can be seen in the section `Zend OPcache`.
+
+    The second file is `opcache.php` (based on [`amnuts/opcache-gui`](https://github.com/amnuts/opcache-gui)). This shows a nicer graphical output of the OPcache status and can be called from the browser via
+
+    - URL = `http://<your.IP>/opcache.php/`
+
+    ![OPcache statistics page](../assets/images/OpCache_php_output.png){: width="700" height="976" loading="lazy"}
+
+    The dialog shows the cache status as well as the settings.
+
+    **I have warnings regarding missing PHP modules (e.g. `bcmath`, `gmp` and `imagick`).**   
+    **What shall I do?**
+
+    Sometimes the Nextcloud settings overview dialog shows a message *"This instance is missing some recommended PHP modules. For improved performance and better compatibility it is highly recommended to install them."*. This issue is described [there in the DietPi forum](https://dietpi.com/phpbb/viewtopic.php?p=27383&sid=c4dbfcb4f4e4d735020aee1f5cfca9ce#p27383), we propose to ignore them, mostly they are not needed.  
+    To get rid of this message anyhow, you can install the mentioned modules via  
+    `sudo apt install php-bcmath php-gmp php-imagick libmagickcore-6.q16-6-extra`.
```

Comment:
The Imagick module info can be muted as well by disabling the Theming app. This can be used for changing the NC blue, logo, text/URL on the login screen, but I guess most users don't make use of it anyway.

Another point is that it would be better to install the versioned module packages, i.e. `php7.4-bcmath` etc, but this makes it complicated here 🤔. I'll check for a possible one-liner when I'm back home, but until then not a stopper, since we do not add 3rd party PHP repositories our end anymore.